Test.txt

```diff
@@ -1 +1,3 @@
-This is a test.
\ No newline at end of file
+This is how you change a file.
```

Comment:
Works pretty good.

---
In reply to: [53845000](https://github.com/kivancmuslu/CFTest/pull/1#discussion_r53845000) [](ancestors = 53845000,53844945,53844845)